runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -409,6 +415,12 @@ public void onChatMessage(ChatMessage event)
 			return;
 		}
 
+		if (Arrays.equals(HESPORI_REGION, client.getMapRegions()) && message.equals(HESPORI_LOOTED_MESSAGE))
```

Comment:
Use `client.getLocalPlayer().getWorldLocation().getRegionID()` and compare against a single region instead of comparing against the full region array.

Reply:
Okay sure thanks